### PR TITLE
feat: Disable Gnome Software auto-updates & lock it from changing

### PR DESCRIPTION
**Pull request by fiftydinar**

### CODE REVIEW

File: usr/etc/dconf/db/local.d/locks/01-ublue-lock
Comment (p5) [inline usr/etc/dconf/db/local.d/locks/01-ublue-lock:1]: I never knew about this `/locks/` directory.  Does it work for all dconf settings?

Comment (fiftydinar) [reply usr/etc/dconf/db/local.d/locks/01-ublue-lock:1]: As far as I tested in my own image, everything worked, including this dconf lock.

Comment (p5) [reply usr/etc/dconf/db/local.d/locks/01-ublue-lock:1]: I trust you that this gnome software stuff works, I was asking in general, can anything be in this `/locks` directory where it prevents changes being made in the OS?  Or is it only select few dconf settings?

Comment (fiftydinar) [reply usr/etc/dconf/db/local.d/locks/01-ublue-lock:1]: Well, I'm not the expert in this regard, so I cannot clarify precisely, but I believe that anything that bluefin sets in dconf, will also work in lock too.

**Review by p5 — APPROVED** [review]

**Comment by p5** [comment]
The failing build is unrelated to changes from this PR, so will merge.  
It's just some F39 package conflicts

**Comment by ArtikusHG** [comment]
There is an issue about this in main: https://github.com/ublue-os/main/issues/194

Maybe we should consider merging this into main, and not just bluefin?

**Pull request merged**